### YamlMime:ManagedReference
items:
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  id: RuntimeBinderException
  children:
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  type: Class
  summary: "C# ランタイム バインダーの動的バインドが処理されるときに発生するエラーを表します。"
  remarks: "この種の例外とは異なります<xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>一方、RuntimeBinderException が、通常のコンパイラ エラーの意味でバインドの失敗を表しますが<xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>自体ランタイム バインダーの問題があるを表します</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException></xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>。"
  syntax:
    content: 'public class RuntimeBinderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  id: '#ctor'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref>クラスです。"
  remarks: "このコンス トラクター、 <xref:System.Exception.Message%2A>&quot;defaultmessagedisplayedbyparameterlessctorwritermustsupply&quot;このメッセージのエラーを説明するシステム提供のメッセージの新しいインスタンスのプロパティは考慮に入れて、現在のシステムのカルチャ</xref:System.Exception.Message%2A>。       次の表は、 <xref:System.InsufficientMemoryException>。</xref:System.InsufficientMemoryException>のインスタンスの初期プロパティ値を示しています。      |プロパティ |値 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|ローカライズされたエラー メッセージ文字列 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> 。"
  syntax:
    content: public RuntimeBinderException ();
    parameters: []
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref>を指定したエラー メッセージを持つクラス。"
  remarks: "次の表は、 <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>。</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>のインスタンスの初期プロパティ値を示しています。      |プロパティ |値 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|指定されたエラー メッセージ文字列`message`|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> 。"
  syntax:
    content: public RuntimeBinderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "例外を説明するメッセージ。 このコンス トラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認するために必要です。"
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref>クラスがシリアル化されたデータです。"
  remarks: "このコンス トラクターは、ストリームで送信された例外オブジェクトを再構成する逆シリアル化中に呼び出されます。"
  syntax:
    content: protected RuntimeBinderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "スローされた例外に関するシリアル化されたオブジェクト データを保持するオブジェクト。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "ソースまたは転送先に関する文脈情報。"
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "新しいインスタンスを初期化、 <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref>を指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を持つクラス。"
  remarks: "前の例外への参照を含める必要があります前の例外の直接の結果としてスローされる例外、<xref:System.Exception.InnerException%2A>プロパティ</xref:System.Exception.InnerException%2A>。 <xref:System.Exception.InnerException%2A>プロパティは、コンス トラクターに渡されたものと同じ値を返しますまたは`null`場合、<xref:System.Exception.InnerException%2A>プロパティがコンス トラクターには、内部例外値を指定していません</xref:System.Exception.InnerException%2A></xref:System.Exception.InnerException%2A>。       次の表は、 <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>。</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>のインスタンスの初期プロパティ値を示しています。      |プロパティ |値 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|指定されたエラー メッセージ文字列`message`|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> 。"
  syntax:
    content: public RuntimeBinderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "例外の原因を説明するエラー メッセージ。"
    - id: innerException
      type: System.Exception
      description: "内部例外が指定されていない場合、現在の例外または null 参照の原因となった例外。"
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException.RuntimeBinderException
